aiogram/api/types/user.py

```diff
@@ -1,11 +1,19 @@
 from __future__ import annotations
 
+import typing
 from typing import Optional
 
-from .base import TelegramObject
+import babel  # type: ignore
```

Comment:
Why `type: ignore` is here?

When you want to remove warnings about untyped packages you will need to add exclusion to `mypy.ini`